addons/mail/static/src/messaging/entity/notification_group_manager/notification_group_manager.js

```diff
@@ -0,0 +1,73 @@
+odoo.define('mail.messaging.entity.NotificationGroupManager', function (require) {
+'use strict';
+
+const { registerNewEntity } = require('mail.messaging.entityCore');
+const { one2many } = require('mail.messaging.EntityField');
+
+function NotificationGroupManagerFactory({ Entity }) {
+
+    class NotificationGroupManager extends Entity {
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
+        computeGroups() {
```

Comment:
couldn't make this as a computed field? Looks complex, but it would be better than manually calling this

Reply:
I think it's too complex at least until we have related dependencies.